src/resources/users.ts

```diff
@@ -213,6 +213,7 @@ export class Users extends Resource<{realm?: string}> {
     urlParamKeys: ['id'],
     payloadKey: 'actions',
     queryParamKeys: ['lifespan', 'redirectUri', 'clientId'],
+    headers: {'content-type': 'application/json'},
```

Comment:
I feel like we should be setting this header for pretty much every request. I believe Axios adds the correct content-type by default, so I don't see why it's not happening here. @edewit WDYT?

Reply:
Axios should automatically set the content type to json if it detects that the body is an object. The body of the executeActionEmail is an array of strings and in my case it's a single string: `["UPDATE_PASSWORD"]`. Could it be that it's not detecting that this array should be sent as JSON? 

Can anyone else reproduce the problem or could it be an issue with my local instance?